pallets/liquid-staking-v2/Cargo.toml

```diff
@@ -8,20 +8,20 @@ version = '3.0.0'
 targets = ['x86_64-unknown-linux-gnu']
 
 [dependencies]
-codec         = { package = 'parity-scale-codec', version = '2.2.0', features = ['max-encoded-len'], default-features = false }
-frame-support = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
-frame-system  = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
-orml-traits   = { git = 'https://github.com/open-web3-stack/open-runtime-module-library.git', default-features = false }
-sp-runtime    = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
-sp-std        = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
-xcm           = { git = 'https://github.com/paritytech/polkadot.git', branch = 'release-v0.9.9', default-features = false }
+codec           = { package = 'parity-scale-codec', version = '2.0.0', features= ["max-encoded-len"], default-features = false }
+frame-support   = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
+frame-system    = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
+orml-traits     = { git = 'https://github.com/open-web3-stack/open-runtime-module-library.git', default-features = false }
+sp-runtime      = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
+sp-std          = { git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v0.9.9', default-features = false }
+xcm             = { git = 'https://github.com/paritytech/polkadot.git', branch = 'release-v0.9.9', default-features = false }
+orml-currencies = { git = 'https://github.com/open-web3-stack/open-runtime-module-library.git', default-features = false }
```

Comment:
we dont need `orml-currencies` and `orml-tokens` IIRC, we just need `MultiCurrencyExtended` from orml-traits

Reply:
we need them due to trait requirement in runtime construction.